pkg/cmd/server/server.go

```diff
@@ -344,11 +338,15 @@ func (s *server) run() error {
 		go s.runProfiler()
 	}
 
+	// TODO(freyjo): Adjust comment as soon as we adjust Velero so that the server process and the CRs
+	//  are allowed to reside in different namespaces.
 	// Since s.namespace, which specifies where backups/restores/schedules/etc. should live,
 	// *could* be different from the namespace where the Velero server pod runs, check to make
 	// sure it exists, and fail fast if it doesn't.
-	if err := s.namespaceExists(s.namespace); err != nil {
-		return err
+	if s.namespace != "" {
```

Comment:
Let's revert this diff as discussed, as s.namespace is something like backup which exists